src/services/produce_api/produceAPI/produce/models.py

```diff
@@ -1,10 +1,11 @@
 from django.db import models
 
 class Produce(models.Model):
-  name = models.CharField(max_length=150)
-  thumbnail = models.ImageField(null=True)
-  description = models.TextField()
-  created_date = models.DateTimeField(auto_now_add=True)
+    name = models.CharField(max_length=150)
+    thumbnail = models.ImageField(null=True)
+    description = models.TextField()
+    price = models.DecimalField(decimal_places=3, max_digits=10, default= 00.0)
```

Comment:
Price field of the produce will accept an object id not direct price information. That will be stored in another api.

Add field id to the object model

Reply:
I have removed this field for now